tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -2585,14 +2589,14 @@ private String handleDateFormatItem(String xpath, String value, boolean showCont
                                                     setBackground("'" + tsfResult + "'"),
                                                     setBackground("'" + relativeDayValue + "'")
                                                 }));
-                examples.add(dtf.format(DATE_SAMPLE));
+                examples.add(dtf.format(DATE_SAMPLE));  // 15698: "short time" -- 2 examples?
             }
 
             return formatExampleList(examples.toArray(new String[0]));
         } else {
             String id = parts.findAttributeValue("dateFormatItem", "id");
             if ("NEW".equals(id) || value == null) {
-                return startItalicSymbol + "n/a" + endItalicSymbol;
+                return startItalicSymbol + "n/a" + endItalicSymbol;  // 15698: doesn't seem relevant--meaning??
```

Comment:
It's a mystery to me as well...

Reply:
The id being "NEW" seems odd to me as well; I just checked the DTD, and it isn't permitted:
```
<!ATTLIST dateFormatItem id CDATA #REQUIRED >
<!--@MATCH:regex/((E|EEEE)?(H|HH|h|hh|Bh)(m|ms|mm|mmss)?(Z|z|zzzz|v|vvvv)?)|(ms|mmss)|((G|GGGGG)?(y|yy|yyyy)((M{1,4}((E|EEEE|cccc)?(d|dd))?)|(w|Q|QQQ|QQQQ))?)|(U(M|MMM)d?)|(M{1,4}(((E|EEEE|cccc)?(d|dd))|W)?)|((E|EEEE)?d)|(E|EEEE)-->
```
So that looks obsolete, and we can probably remove just that check.

I wouldn't remove the value==null check, however, without verifying that it can't occur at this point in the code.